ckan/logic/action/create.py

```diff
@@ -619,6 +619,77 @@ def member_create(context, data_dict=None):
     return model_dictize.member_dictize(member, context)
 
 
+def package_member_create(context, data_dict):
```

Comment:
apis are pretty hard to rename, should we go with `package_collaborator_create` et.al instead?

Reply:
Sounds good!
I didn't rename the models (`PackageMember`) because there are not used widely and we might want to rename the `Package` bit at some point anyway :)